src/cli/print/print.go

```diff
@@ -89,6 +89,39 @@ func err(w io.Writer, s ...any) {
 	fmt.Fprint(w, msg...)
 }
 
+// Print prints the params to cobra's output writer (stdOut by default)
+// if s is nil, prints nothing.
+func Print(ctx context.Context, s ...any) {
+	print(getRootCmd(ctx).OutOrStdout(), s...)
+}
+
+// print is the testable core of Print()
+//revive:disable:redefines-builtin-id
+func print(w io.Writer, s ...any) {
```

Comment:
Lets not redefine the builtin here (I recognize we were already doing that; should've renamed it then, too).  It's an unexported func, it doesn't need to comply with the other patterns exactly.